### PR TITLE
WIP: Add queries hash to cache id.

### PR DESCRIPTION
**Pull request by pmelab**

### CODE REVIEW

File: src/EventSubscriber/CacheSubscriber.php [hunk 1]
```diff
@@ -237,7 +238,9 @@ public function onResponse(FilterResponseEvent $event) {
     $expire = $this->maxAgeToExpire($metadata->getCacheMaxAge());
 
     // Write the cache entry for the cache metadata.
-    $ccid = $this->getCacheIdentifier($this->metadata);
+
+    $queryHash = md5(serialize($request->attributes->get('queries')));
```
Comment (fubhy) [inline src/EventSubscriber/CacheSubscriber.php:242]: This should be a cache context instead, no? That's the whole point of cache contexts ;)

**Review by fubhy — CHANGES_REQUESTED** [review]
Instead of building the cache id manually, you should be using cache contexts.

**Comment by pmelab** [comment]
Yes, this is just a quick fix we needed. Pushed it for reference.

**Comment by pmelab** [comment]
This has been solved by #306.

**Pull request closed without merging**